plugins/application/prometheus/main.go

```diff
@@ -168,6 +174,7 @@ func (pc *PromCollector) UpdateMetrics(name string, time float64, typ data.Metri
 	cacheKey := pc.cacheindexbuilder.String()
 	mProcItf, found := pc.mProc.Load(cacheKey)
 	if !found {
+		// pc.logger.Infof("creating new metric with cache key '%s'", cacheKey)
```

Comment:
Is this something you no longer need, or a comment showing how to use this? If this is code that is no longer being used, please remove vs commenting out.

Reply:
deleting it